src/LogLambda/index.ts

```diff
@@ -30,6 +37,7 @@ const events = {
   'CodePipeline Pipeline Execution State Change': {
     shouldTriggerAlert: () => true,
     formatter: CodePipelineFormatter,
+    low_priority: true,
```

Comment:
Is het een idee om de 'ECS Task State Change' ook low_priority te maken?

Reply:
Ja daar twijfelde ik over, ongepland wil je wel weten, maar dan krijg je vermoedelijk ook wel andere alarmen.